gcode_metadata/metadata.py

```diff
@@ -237,14 +240,34 @@ def cache_name(self):
         return new_path
 
     def is_cache_fresh(self):
-        """If cache is fresher than file, returns True"""
+        """Checks if we can use the current cache file"""
+        return self.is_cache_recent() and self.is_cache_correct_version()
+
+    def is_cache_recent(self):
+        """Checks if the cache file is newer than the source file"""
         try:
             file_time_created = os.path.getctime(self.path)
             cache_time_created = os.path.getctime(self.cache_name)
-            return file_time_created < cache_time_created
         except FileNotFoundError:
             return False
 
+        if file_time_created > cache_time_created:
```

Comment:
Ten for je v tom že to rovnou vrátí ten výraz
```
return a < b
```
Nějak nechápu, co ti na tom jednom řádku vadí... a skoro bych čekal, že ti na tomhle bude křičet linter, že to máš vrátit jako výraz